src/main/resources/scripts/sequence-files/purge_sequence_files.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python
+import argparse
+import mysql.connector
+import os
+
+def remove(path, purge):
+    if purge:
+        try:
+            if os.path.isdir(path):
+                os.rmdir(path)
+            elif os.path.isfile(path):
+                os.remove(path)
+            else:
+                print("Unable to delete ", path)
+        except OSError as e:
+            print(e)
+    else:
+        print(path)
+
+def list_sequence_files(host, user, password, database):
+    db = mysql.connector.connect(
+        host=host,
+        user=user,
+        password=password,
+        database=database
+    )
+    cursor = db.cursor()
+    # TODO: Should we double check this file doesn't exist in the actual table in case it was manually restored?
+    cursor.execute("SELECT DISTINCT file_path FROM sequence_file_AUD WHERE revtype=2")
+    result = cursor.fetchall()
+    cursor.close()
+    db.close()
+    return result
+
+def main():
+    parser = argparse.ArgumentParser(description="This program lists the sequence files and folders that have been previously deleted in IRIDA.")
+    parser.add_argument('--purge', help="Deletes the sequence files and folders from the filesystem.", action="store_true")
+    parser.add_argument('--baseDirectory', help="The sequence file base directory.", required=True)
```

Comment:
Please add a line to the help saying something like "the folder is usually formatted like `.../irida/sequence-files/`
I wasn't sure if I just needed to point it to /tmp/irida/ or where when I used it the first time.

Reply:
I added a default value for the sequence directory. I hope that's okay. Please see 1b0070c416a6388551174d90069a43ee946862eb.